### DISCUSSION TITLE
GHCR image

### DISCUSSION THREAD

**Discussion by TMTwatch** [a-sync/game-server-watcher#19] [Ideas / requests]
Can you guys make a build for the GitHub container registry? This way, Game Server Watcher can be supported by TrueCharts.

**Comment by a-sync** [comment]
hey,
does it need to be a standalone image? maybe use the Dockerfile in the repo? (gsw Dockerfile uses the standard nodejs docker image)  

guess i could add a github action that releases an image to ghcr using the existing Dockerfile.  
i will look into this and update here with any progress.

**Comment by a-sync** [comment]
So i added a workflow to release docker images to ghcr whenever a `v*.*.*` tag is pushed.
gave it a test run with v3.0.0 and it created the image, which i tested with the existing compose file and seems to work fine.

You can find the image repo here: `ghcr.io/a-sync/game-server-watcher:latest`

Beware that the current image (v3.0.0) is built from the gamedig-next branch. I will push a new tag/release once gamedig v5 is finalized and the gamedig-next changes are merged into the master branch.

**Reply by TMTwatch** [reply]
cool. I'll relay the message and get back to you asap

**Comment by TMTwatch** [comment]
[I think that worked](https://github.com/truecharts/charts/pull/18718) 👍
